public-config.yml

```diff
@@ -5,18 +5,18 @@ machete-git-uri: git@github.com:cloudfoundry/machete.git
 buildpack-packager-git-uri-public: https://github.com/cloudfoundry/buildpack-packager
 buildpack-packager-git-uri: git@github.com:cloudfoundry/buildpack-packager.git
 
-binary-builder-git-uri: https://github.com/cloudfoundry/binary-builder.git
+binary-builder-git-uri: https://github.com/SUSE/cf-binary-builder.git
 
 cf-release-git-uri: https://github.com/cloudfoundry/cf-release
 
-buildpacks-github-org: cloudfoundry
+buildpacks-github-org: SUSE
 buildpacks-binaries-s3-bucket: pivotal-buildpacks
 bosh-lite-domain-name: cf-app.com
 
-buildpacks-ci-pipeline-uri: https://buildpacks.ci.cf-app.com/teams/main/pipelines/buildpacks-ci
-buildpacks-ci-git-uri-public: https://github.com/cloudfoundry/buildpacks-ci
-public-buildpacks-ci-robots-git-uri-public: https://github.com/cloudfoundry/public-buildpacks-ci-robots
-buildpacks-ci-git-uri: git@github.com:cloudfoundry/buildpacks-ci.git
+buildpacks-ci-pipeline-uri: http://concourse.suse.de/teams/main/pipelines/cf-buildpacks-ci
```

Comment:
Ah, we're using that concourse (instead of the office-local one)? I guess it makes sense (since that has wider visibility)?